_dashboards/management/accelerate-external-data.md

```diff
@@ -14,35 +13,172 @@ Introduced 2.11
 
-A _covering index_ ingests all or some of the data from the source into OpenSearch and makes it possible to use all OpenSearch Dashboards and plugin functionality. See the [Flint Index Reference Manual](https://github.com/opensearch-project/opensearch-spark/blob/main/docs/index.md) for comprehensive guidance on this feature's indexing process.
+For comprehensive guidance on each indexing process, see the [Flint Index Reference Manual](https://github.com/opensearch-project/opensearch-spark/blob/main/docs/index.md) 
 
 ## Data sources use case: Accelerate performance
 
-To get started with the **Accelerate performance** use case available in **Data sources**, follow these steps:
+To get started with accelerating query performance, perform the following steps:
 
-1. Go to **OpenSearch Dashboards** > **Query Workbench** and select your Amazon S3 data source from the **Data sources** dropdown menu in the upper-left corner.
-2. From the left-side navigation menu, select a database.
-3. View the results in the table and confirm that you have the desired data.
+1. Go to **OpenSearch Dashboards** > **Query Workbench** and select your data source from the **Data sources** dropdown menu.
+2. From the navigation menu, select a database.
+3. View the results in the table and confirm that you have the correct data.
 4. Create an OpenSearch index by following these steps:
-    1. Select the **Accelerate data** button. A pop-up window appears. 
-    2. Enter your details in **Select data fields**. In the **Database** field, select the desired acceleration index: **Skipping index** or **Covering index**. A _skipping index_ uses skip acceleration methods, such as partition, min/max, and value sets, to ingest data using compact aggregate data structures. This makes them an economical option for direct querying scenarios. A _covering index_ ingests all or some of the data from the source into OpenSearch and makes it possible to use all OpenSearch Dashboards and plugin functionality.
-5. Under **Index settings**, enter the information for your acceleration index. For information about naming, select **Help**. Note that an Amazon S3 table can only have one skipping index at a time.
+    1. Select **Accelerate data**. A pop-up window appears. 
+    2. Enter your database and table details under **Select data fields**.
+5. For **Acceleration type**, select the type of acceleration according to your use case. For more information, see the following sections:
+      - [Skipping indexes](https://opensearch.org/docs/latest/dashboards/management/accelerate-external-data/#skipping-indexes)
+      - [Covering indexes](https://opensearch.org/docs/latest/dashboards/management/accelerate-external-data/#covering-indexes)
+      - [Materialized views](https://opensearch.org/docs/latest/dashboards/management/accelerate-external-data/#materialized-views)
+6. Under **Define index**, enter the information for your acceleration type.
```

Comment:
Is there a save step after this? Similar to `View the covering index query details and then click **Run**. OpenSearch adds your index to the left navigation pane.`

Reply:
This section is intended to get users started with the initial steps, and then link off to the next three sections to finish out the process. I rewrote it a bit and it looks better now.